src/Jumbotron.js

```diff
@@ -1,31 +1,37 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import elementType from 'prop-types-extra/lib/elementType';
 
-import { bsClass, getClassSet, splitBsProps } from './utils/bootstrapUtils';
+import { createBootstrapComponent } from './ThemeProvider';
 
 const propTypes = {
-  componentClass: elementType
+  componentClass: elementType,
+  fluid: PropTypes.bool
 };
 
 const defaultProps = {
-  componentClass: 'div'
+  componentClass: 'div',
+  fluid: false
 };
 
 class Jumbotron extends React.Component {
   render() {
-    const { componentClass: Component, className, ...props } = this.props;
-    const [bsProps, elementProps] = splitBsProps(props);
-
-    const classes = getClassSet(bsProps);
-
-    return (
-      <Component {...elementProps} className={classNames(className, classes)} />
-    );
+    const {
+      componentClass: Component,
+      className,
+      fluid,
+      ...props
+    } = this.props;
+    const classes = {
+      jumbotron: true,
```

Comment:
instead of a literal `'jumbotron'` here, we can now use `props.bsPrefix` which is passed in by the `createBootstrapComponent` so you can write the classes like: the below (after spreading out bsPrefix in the props block above)
```js
{
  [bsPrefix]: true,
  [`${bsPrefix}-fluid`]: fluid
}
```

Reply:
we also want to add a propType in for it with a comment similar to: https://github.com/react-bootstrap/react-bootstrap/blob/v4-support/src/ButtonGroup.js#L13

Reply:
the `[]` are unnecessary here. No?